src/backend/joanie/core/models/products.py

```diff
@@ -274,14 +274,21 @@ def validate(self):
         """
         if self.state == enums.ORDER_STATE_VALIDATED:
```

Comment:
The code would look nicer (less indentation) by eliminating the opposite case first:
```python
if self.state != enums.ORDER_STATE_VALIDATED:
    return
```